### ISSUE TITLE
wikipedia page for ODL

### ISSUE THREAD

**Issue by mehrhardt** [odlgroup/odl#1346]
Not sure about this myself but it might be worth thinking about whether ODL should get a wikipedia page, see https://en.wikipedia.org/wiki/ODL.

**Comment by adler-j** [comment]
Great Idea. Would you like to start one? The rest of us could help filling it in.

**Comment by aringh** [comment]
Short draft created. Anyone can edit, and within six months we need to submit for review in order to publish is. https://en.wikipedia.org/wiki/Draft:Operator_Discretization_Library

**Comment by mehrhardt** [comment]
I think this is already pretty good! We could already publish it like this and keep changing it from time to time.

Additional information that could be included: developers, fields / applications this has already been used for, more details on how it works / why it is good.

**Comment by adler-j** [comment]
I added a nice little software sidebar to it

**Comment by aringh** [comment]
I have never edited wiki-articles before. Before creating the draft I had to add a COI-disclaimer on my personal page. Does anyone know how these work exactly? Can I state what kind of conflict of interest it is (contributor to the open source project)?

**Comment by aringh** [comment]
Also, I think that @ozanoktem should have a look at it before we submit :wink:

**Comment by sbanert** [comment]
I don’t know what applies to the English Wikipedia, but in the German one the article would be deleted almost instantly because it violates the relevance criteria (for the English equivalent see https://en.wikipedia.org/wiki/Wikipedia:Notability).

**Comment by adler-j** [comment]
I kinda agree with @sbanert. I guess a good reference for something that is accepted as relevant is [https://en.wikipedia.org/wiki/FEniCS_Project](FEniCS), which first got a wikipedia page in 2010.

**Comment by mehrhardt** [comment]
I just noticed that the draft https://en.wikipedia.org/wiki/Draft:Operator_Discretization_Library got deleted. What happened?

**Comment by adler-j** [comment]
I guess the drafts timeout after 6 months as per https://en.wikipedia.org/wiki/Wikipedia:Drafts

with that said ODL is getting mature enough to actually have a wikipedia page by now.

**Comment by mehrhardt** [comment]
Agreed. Does anyone have a copy of what we wrote last year?

**Comment by aringh** [comment]
... :disappointed: You can request undeletion: https://en.wikipedia.org/wiki/Wikipedia:Requests_for_undeletion/G13 Should we do that and try to publish the draft?

**Comment by mehrhardt** [comment]
I requested an undeletion and now it is back: https://en.wikipedia.org/wiki/Draft:Operator_Discretization_Library

We should be finalising this soon so that it does not disappear again!

What do you think is missing / needed?

**Comment by adler-j** [comment]
The history part could use a source and perhaps the list of places involved could be reduced? Otherwise, I think we could perhaps have a go for it.

**Comment by aringh** [comment]
Does the github page work as a source? Quoting:

> **ODL developers**
Development of ODL started in 2014 as part of the project "Low complexity image reconstruction in medical imaging” by Ozan Öktem (@ozanoktem), Jonas Adler (@adler-j) and Holger Kohr (@kohr-h). Several others have made significant contributions, see the contributors list.
>
>To contact the developers either open an issue on the issue tracker or send an email to odl@math.kth.se.
>
>**Funding**
ODL has primarily been developed at KTH Royal Institute of Technology, Stockholm and Centrum Wiskunde & Informatica (CWI), Amsterdam. It is financially supported by the Swedish Foundation for Strategic Research as part of the project "Low complexity image reconstruction in medical imaging".
>
>Some development time has also been financed by Elekta.

**Comment by aringh** [comment]
Should we just submit the draft, before it expires again?

**Comment by adler-j** [comment]
Yeah, we should simply quote the github page. Then we can submit it

**Comment by aringh** [comment]
We got the submission declined due to ``not adequately supported by reliable sources`` :disappointed: Should we cite some of the articles citing ODL, and/or Zenodo?

**Comment by mehrhardt** [comment]
Good idea, @aringh!

What is the response from wikipedia as a whole? (i.e. putting the quote into context)

**Comment by aringh** [comment]
Actually, that is basically it: ``This submission is not adequately supported by reliable sources. Reliable sources are required so that information can be verified. If you need help with referencing, please see Referencing for beginners and Citing sources.``

The response as a whole can be found on the [draft page](https://en.wikipedia.org/wiki/Draft:Operator_Discretization_Library).

**Comment by aringh** [comment]
I have added an number of references. Please feel free to have a look and update. If no one updates or otherwise comments on it, I will resubmit in a while.

**Comment by adler-j** [comment]
Looks nice to me!

**Comment by aringh** [comment]
This has been turned down again, see the [draft page](https://en.wikipedia.org/wiki/Draft:Operator_Discretization_Library). Not sure what to do about it...

**Comment by mehrhardt** [comment]
Same here. Not sure what coverage is needed to justify a wikipedia article. I just noted that even ASTRA does not have one, not sure they tried though.

**Comment by adler-j** [comment]
It might be that we simply have to wait with a Wikipedia page at this point and not keep pushing it. I'll leave this issue open for now though.
